### ISSUE TITLE
accessibility: Table of Contents should be links, not buttons

### ISSUE THREAD

**Issue by openstaxalina** [Connexions/webview#1944]
### Problem
For some reason, the software for the screen reader is not expanding Chapters to their sections in the navigation pane when you press enter. We understand it is because they're programmed as buttons instead of links. 
...
Navigation is difficult, as a lot of the table of contents navigation panel is coded using expandable “buttons” or “span” (not links). Screen readers won’t display these links correctly when they bring up their shortcut links menu.

### More Detail
From Phil: 

I think there are 2 problems:

1. the twisty is CSS and on a < span > which does not show up for screenreaders (it should be a link or button or something rather than a CSS ::before
2. the big blue [Contents + ] button should have aria labels that indicate something like "Click to Show the Table of Contents"

**Comment by openstaxalina** [comment]
![image 2](https://user-images.githubusercontent.com/7863733/45575495-58f68e00-b839-11e8-81bb-20df5fbc83a0.png)
![image 3](https://user-images.githubusercontent.com/7863733/45575497-5ac05180-b839-11e8-90f1-250239db8383.png)

**Comment by PiotrKozlowski** [comment]
Ready for testing in #1967

**Comment by uskka** [comment]
@openstaxalina Works on default text to speech app on Ubuntu, doesn't work with Chromevox on Mac and Windows, also doesn't work with VoiceOver on Mac.

**Comment by openstaxalina** [comment]
@uskka and @PiotrKozlowski thank you!

@PiotrKozlowski, do you know of any strategies that would make this functional on a broader set of screen readers?  I will also ask our accessibility group about this on Thursday.

**Comment by PiotrKozlowski** [comment]
@openstaxalina I will try to fix this today.

**Comment by uskka** [comment]
In Chromevox it reads only one random chapter, in voiceover on Mac it doesn't work at all.

**Comment by PiotrKozlowski** [comment]
Hmm, for me it's working in incognito mode. My guess is Chromevox is not reading automatically links in lists which was already readed.

**Comment by PiotrKozlowski** [comment]
Ready for testing at master

**Comment by uskka** [comment]
@openstaxalina Looks good, but it reads only the first section when the list with section titles is opened. I attached a recording from the test. Could you please confirm that this is a desired behaviour for this issue? Recorded with Chromevox. 
https://drive.google.com/open?id=1UvjlRTSSxNPYMXVIy47FAma0NpdZHDH6

**Comment by PiotrKozlowski** [comment]
It basically reads how many elements is in the list and after this it reads the current list item and for nested lists is saying how many element is in the main list, then its saying that you are currently in one of the list item and that this item is also a list with X items and after this it reads current list item.

**Comment by openstaxalina** [comment]
@uskka when you have Chapter 3 expanded, can you use Chromevox to navigate *down* to get to 3.1?

**Comment by uskka** [comment]
@openstaxalina Of course I can, every time I press tab it goes down by 1 position.

**Comment by PiotrKozlowski** [comment]
And shift+tab goes up

**Comment by openstaxalina** [comment]
Okay, great - I think this is ready to go!
